src/content/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/anomaly-detection-applied-intelligence.mdx

```diff
@@ -12,7 +12,7 @@ redirects:
   - /docs/alerts-applied-intelligence/applied-intelligence/proactive-detection/proactive-detection-applied-intelligence
 ---
 
-With applied intelligence's proactive detection, anomalies from your APM-monitored applications are automatically surfaced in our activity streams and anomalies feed. You can click each anomaly to bring up an automatic analysis.
+With applied intelligence's anomlay detection, anomalies from your APM-monitored applications are automatically surfaced in our activity streams and anomalies feed. You can click each anomaly to bring up an automatic analysis.
```

Comment:
Do Not change "proactive detection" to "anomaly detection". We'll do a separate effort to update the documentation on proactive detection.  Please leave all proactive detection files as is. This is a separate product.

Reply:
fixed!